egs2/voxtlm/lm1/local/text_normalize.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+. path.sh
+
+
+# check if normalize-punctuation.perl exists
+if ! command -v <normalize-punctuation.perl> &> /dev/null
+then
+    echo "normalize-punctuation.perl could not be found. Compile moses in espnet/tools as make moses.done"
+    exit 1
+fi
+
+# remove punctuations from text
+if [ "$#" -ne 1 ]; then
+    echo "Illegal number of parameters"
+fi
+
+text_dir=$1
+
```

Comment:
I recommend you show an example before/after this normalization here.

Reply:
Added example.